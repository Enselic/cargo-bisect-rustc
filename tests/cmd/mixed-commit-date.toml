bin.name = "cargo-bisect-rustc"
args = "--start 6cbf0926d54c80ea6d15df333be9281f65bbeb36 --end 2024-03-05"
status = "failed"
