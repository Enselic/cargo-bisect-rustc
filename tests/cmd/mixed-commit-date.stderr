ERROR: cannot take different types of bounds for start/end, got start: Some(Commit("6cbf0926d54c80ea6d15df333be9281f65bbeb36")) and end Some(Date(2024-03-05))
